.github/workflows/tests.yaml

```diff
@@ -13,7 +13,11 @@ jobs:
     env:
       # This is not a secret because it is a free API key. At worst,
       # we may hit the rate limit.
-      EIA_API_KEY: "9e1a7ae3ce839e9ad174eb4028b7a7b2"
+      EIA_API_KEY: ${{ secrets.EIA_API_KEY }}
```

Comment:
We need to fill in any of these that are missing (I can't see the repo secrets).